.gitignore

```diff
@@ -10,3 +10,4 @@ npm-debug.log
 # other
 .next
 .DS_Store
+.idea
```

Comment:
I would prefer if you could add this to your [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) 

Otherwise this list will just keep on growing 👍